readme.adoc

```diff
@@ -583,7 +583,20 @@ Make sure `kubectl` CLI is installed and configured for the Kubernetes cluster.
 
 === Deployment: Kubepack
 
-https://github.com/aws-samples/aws-microservices-deploy-options/issues/63
+Make sure `kubectl` and `kubepack` CLI is installed and configured for the Kubernetes cluster. To install `kubepack` cli, run:
+
+```
+wget -O kubepack https://github.com/kubepack/kubepack/releases/download/0.1.0/kubepack-darwin-amd64 \
+  && chmod +x kubepack \
+  && sudo mv kubepack /usr/local/bin/
+```
+
+. Move to package root directory: `cd apps/k8s/kubepack`
+. Pull dependent packages: `pack dep -f .`
```

Comment:
Downloaded binary name is `kubepack` but the command uses `pack`. It needs to be consistent.